src/Elyon.Fastly.Api.PostgresRepositories/TestingPersonnelInvitationsRepository.cs

```diff
@@ -50,6 +51,31 @@ public async Task<Guid> GetInvitationIdByDateAsync(DateTime testDate)
                 .ConfigureAwait(false);
         }
 
+        public async Task IncreaseShiftCountAsync(Guid invitationId, ShiftNumber shiftNumber, int capacityToAdd)
+        {
+            await using var context = ContextFactory.CreateDataContext(null);
+            var items = context.TestingPersonnelInvitations;
+            var entity = await items.AsNoTracking()
+                .FirstAsync(t => t.Id == invitationId)
+                .ConfigureAwait(false);
+
+            if (shiftNumber == ShiftNumber.First)
+            {
+                entity.RequiredPersonnelCountShift1 += capacityToAdd;
+            }
+
+            if (shiftNumber == ShiftNumber.Second)
```

Comment:
you can make it if/else if so that we can skip the second check if we enter the first one